src/index.html

```diff
@@ -7,8 +7,25 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap&subset=cyrillic" rel="stylesheet">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+    <a class="header-logo" href="#">
+      <img src="images/logo.png" alt="MOYO" class="logo">
+    </a>
+    <nav class="nav">
+      <ul class="nav-list">
+        <li class="link-item is-active"><a href="#" class="nav-link" >APPLE</a></li>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.